substrate/frame/support/procedural/src/construct_runtime/expand/metadata.rs

```diff
@@ -58,7 +58,12 @@ pub fn expand_runtime_metadata(
 					#attr
 				}
 			});
+			let deprecation_info = {
```

Comment:
nit: We can move this below into an `exapnd_pallet_metadata_deprecation`

Reply:
done